_includes/setup/instructions/floodgate/viaproxy.md

```diff
@@ -0,0 +1,14 @@
+# Floodgate setup with Geyser for ViaProxy
+
+<div class="alert alert-info" role="alert">
+	To be able to use Floodgate authentication, you need to be able to install Floodgate on the Java server you are connecting to.
+</div>
+
+1. Download and install the platform-specific Floodgate jar (e.g. floodgate-spigot, or floodgate-fabric) [here](https://geysermc.org/download). 
```

Comment:
Please update this link, as done [here](https://github.com/GeyserMC/GeyserWiki/commit/b395c4591b069c2dfff233e7c41276121a187431)